apps/website/src/pages/animal-quest/[episodeName].tsx

```diff
@@ -167,6 +168,11 @@ export const getStaticProps: GetStaticProps<
   };
 };
 
+const posterUrl = /^http:\/\/localhost(:\d+)?$/.test(env.NEXT_PUBLIC_BASE_URL)
+  ? "https://files.alveus.site/intros/animalquest.png"
```

Comment:
I'm not sure I'm overly a fan of this, but a valid URL of some sort is needed here otherwise Cloudflare errors out (localhost is not seen as a valid URL).

Reply:
I've filed https://github.com/cloudflare/stream-react/issues/106 + https://github.com/cloudflare/stream-react/issues/105 which might make this easier